frontends/p4/toP4/toP4.cpp

```diff
@@ -870,7 +870,8 @@ bool ToP4::preorder(const IR::NamedExpression* e) {
     return false;
 }
 
-bool ToP4::preorder(const IR::StructInitializerExpression* e) {
+bool ToP4::preorder(const IR::StructExpression* e) {
+    builder.append("(");
```

Comment:
Not worth trying to figure out when the parens are necessary from the context and only output them if so?

Reply:
Actually, I think they are only needed when `expressionPrecedence > Prec_Prefix`?